### PR TITLE
Updating AccountingSettings to use composition

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by kelsos** [comment]
@lukicenturi code should be finalized, I only need to check for any failing tests

File: frontend/app/src/components/defi/loan/loans/AaveLending.vue [hunk 1]
```diff
@@ -52,19 +58,25 @@ export default defineComponent({
...
-  mixins: [AssetMixin],
   props: {
     loan: {
       required: true,
       type: Object as PropType<AaveLoan>
     }
   },
-  setup() {
+  setup(props) {
+    let { loan } = toRefs(props);
```
Comment (lukicenturi) [inline frontend/app/src/components/defi/loan/loans/AaveLending.vue:68]: Is there any reason why to use `let` instead of `const`?

Comment (kelsos) [reply frontend/app/src/components/defi/loan/loans/AaveLending.vue:68]: nice catch, no, but it seems that the linter didn't catch it for some reason

File: frontend/app/src/composables/settings.ts [hunk 2]
```diff
@@ -82,3 +88,86 @@ export const setupSettings = () => {
...
+    set(success, formatMessage(i18n.tc('settings.saved'), message));
+  };
+
+  const setError = (message: string) => {
+    set(error, formatMessage(i18n.tc('settings.not_saved'), message));
+  };
+
+  const wait = async () => await promiseTimeout(200);
+  const { start, stop } = useTimeoutFn(clear, 5500);
+  watch([error, success], () => start());
```
Comment (kelsos) [inline frontend/app/src/composables/settings.ts:162]: Hmm I need to change that, wait a moment

Comment (kelsos) [reply frontend/app/src/composables/settings.ts]: OK changed it to only start if there is either an error or success message.

**Pull request merged**